src/parsers/manifest/dash/common/parse_representation_index.ts

```diff
@@ -147,10 +145,12 @@ export default function parseRepresentationIndex(
                    ...segmentTemplates as [
                      ISegmentTemplateIntermediateRepresentation
                    ] /* Ugly TS Hack */);
+    context.availabilityTimeComplete =
```

Comment:
What about availabilityTimeComplete coming from BaseURL elements?

Reply:
I re-checked, those seem to be always ignored for now in profit of those present at the `Period`, `Adaptation`, `Representation` and/or `SegmentTemplate` level.

I tried to implement right now when found on a `BaseURL`, but I found that this implementation could be ambiguous (what if there's both at the Period/Adaptation/Representation/SegmentTemplate level and at the BaseURL level?), so I try to refer to the DASH-IF draft, but I see no mention of it at the BaseURL level (though I'm sure we've seen it somewhere in the past).
All tested streams also put it on either the Period/Adaptation/Representation/SegmentTemplate level.
 So for now, I'm just ignoring `availabilityTimeComplete` present at the `BaseURL` level.